nucypher/characters/base.py

```diff
@@ -110,7 +110,7 @@ def __init__(self,
         #
         # Operating Mode
         #
-        if federated_only is False and BlockchainInterface is None:
+        if not federated_only and blockchain is None:
```

Comment:
:+1: good eye!

Reply:
@szotov and I have been calling this "The sky is falling"